grunt-tasks/jst.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function(grunt) {
+  grunt.config("jst", {
+    compile: {
+      options: {
+        namespace: "wcTmpl",
+        prettify: true,
+        processContent: function(src) {
+          // strip the opening and closing <script> tags...
+          // otherwise, the template functions will just inject script elements
+          // that won't render.
+          src = src.split(/\<script type=['"]text\/template['"]>/)[1];
```

Comment:
switch to backend rendering 😄  Wrong answer?
Honestly I'm not sure what we are trying to do in this pull request. I'm missing a bit of context, why we need to abandon jinja templates for more jsification (or more exactly how I perceive it after being two weeks into the woods 🌳 ahah)